src/data/missions.csv

```diff
@@ -948,32 +948,32 @@
 "ユメヲカケル！を\n見よう","Watch 'Yume wo kakeru!'"
 "特定の編成でユメヲカケル！を\n見よう","Watch 'Yume wo kakeru! (special version)'"
 "涙ひかって明日になれ！を\n見よう","Watch 'Namida hikatte ashita ni nare!'"
-"BLAZEを\n見よう",""
+"BLAZEを\n見よう","Watch 'BLAZE'"
 "累計フォロー数25人を達成しよう","Achieve a cumulative total of 25 followers"
 "累計フォロー数50人を達成しよう","Achieve a cumulative total of 50 followers"
 "累計フォロワー数100人を達成しよう","Achieve a cumulative total of 100 followers"
 "累計フォロワー数200人を達成しよう","Achieve a cumulative total of 200 followers"
 "代表ウマ娘を他のトレーナーに\n育成で100回使ってもらおう","Have other trainers use your horse girl in training 100 times"
-"育成でURAファイナルズを\n大差勝ちで勝利しよう",""
-"ファン数を100万以上にして育成を完了しよう",""
+"育成でURAファイナルズを\n大差勝ちで勝利しよう","Win URA Finals by a large margin"
+"ファン数を100万以上にして育成を完了しよう","Complete a training run with over 1 million fans"
 "育成で河原に30回でかけよう",""
-"育成で短距離適性[F]以下の\n育成ウマ娘で短距離の重賞に勝利しよう",""
-"育成でマイル適性[F]以下の\n育成ウマ娘でマイルの重賞に勝利しよう",""
-"育成で中距離適性[F]以下の\n育成ウマ娘で中距離の重賞に勝利しよう",""
-"育成で長距離適性[F]以下の\n育成ウマ娘で長距離の重賞に勝利しよう",""
-"育成でダート適性[F]以下の\n育成ウマ娘でダートの重賞に勝利しよう",""
-"育成でバ場適正、距離適性、脚質適性を\nそれぞれ[S]に継承させてレースに勝利しよう",""
+"育成で短距離適性[F]以下の\n育成ウマ娘で短距離の重賞に勝利しよう","Win a big Short race with a girl that has F or below Short aptitude"
+"育成でマイル適性[F]以下の\n育成ウマ娘でマイルの重賞に勝利しよう","Win a big Mile race with a girl that has F or below Mile aptitude"
+"育成で中距離適性[F]以下の\n育成ウマ娘で中距離の重賞に勝利しよう","Win a big Medium race with a girl that has F or below Medium aptitude"
+"育成で長距離適性[F]以下の\n育成ウマ娘で長距離の重賞に勝利しよう","Win a big Long race with a girl that has F or below Long aptitude"
+"育成でダート適性[F]以下の\n育成ウマ娘でダートの重賞に勝利しよう","Win a big dirt race with a girl that has F or below Dirt aptitude"
+"育成でバ場適正、距離適性、脚質適性を\nそれぞれ[S]に継承させてレースに勝利しよう","Win a race with track, distance and strategy aptitude S"
 "累積100万マニーためよう","Earn a cumulative 1 million total money"
 "累積1000万マニーためよう","Earn a cumulative 10 million total money"
 "クレーンゲームでぬいぐるみを\n10個以上獲得してクリアしよう","Win 10 or more stuffed toys in the Crane Game"
-"クレーンゲームで\nぬいぐるみを300個獲得しよう",""
+"クレーンゲームで\nぬいぐるみを300個獲得しよう","Win a cumulative 300 stuffed toys in the Crane Game"
 "ストーリーを\n100種類閲覧しよう","Read 100 stories"
 "メインストーリー第1章を\nすべて見よう","Read all of Main Story's chapter 1"
 "メインストーリー第2章を\nすべて見よう",""
 "メインストーリー第3章を\nすべて見よう",""
-"メインストーリー第4章を\nすべて見よう",""
+"メインストーリー第4章を\nすべて見よう","Read the entire chapter 4 of the main story"
```

Comment:
Let's make these uniform

Reply:
They're made uniform in the currently open PR, i didn't change it to avoid more conflicts